[Bug]: Table import and export buttons are broken due to missing classic icons.

### Describe the Bug

When using the Classic icon theme, the table import and export buttons do nothing. The buttons work fine when using the Rod Takehara icon theme.

### To Reproduce

1. Go to Edit > Preferences, Themes tab, change **Icon Theme** to "Classic"
2. Restart MapTool
3. Open the Tables window
4. Click the "import" button. Observe nothing happens.
5. Create a new table and select it.
6. Click the "export" button. Observe nothing happens.

### Expected Behaviour

Table import and export should function regardless of icon theme.

### Screenshots

_No response_

### MapTool Info

MapTool 1.13.2

### Desktop

Linux Mint 21.2

### Additional Context

Here is the exception logged at startup:
<details>
<pre>
2023-08-30 08:54:59.692 ERROR net.rptools.maptool.client.swing.AbeillePanel - Could not init method: initImportButton
java.lang.reflect.InvocationTargetException: null
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at net.rptools.maptool.client.swing.AbeillePanel.panelInit(AbeillePanel.java:99) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.lookuptable.LookupTablePanel.<init>(LookupTablePanel.java:48) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.getLookupTablePanel(MapToolFrame.java:730) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.initializeFrames(MapToolFrame.java:651) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.configureDocking(MapToolFrame.java:540) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.<init>(MapToolFrame.java:451) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.MapTool.initialize(MapTool.java:665) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.MapTool.lambda$main$9(MapTool.java:1788) ~[MapTool.jar:1.13.2]
        at java.awt.event.InvocationEvent.dispatch(Unknown Source) ~[?:?]
        at java.awt.EventQueue.dispatchEventImpl(Unknown Source) ~[?:?]
        at java.awt.EventQueue$4.run(Unknown Source) ~[?:?]
        at java.awt.EventQueue$4.run(Unknown Source) ~[?:?]
        at java.security.AccessController.doPrivileged(Unknown Source) [?:?]
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:?]
        at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:?]
        at net.rptools.maptool.client.swing.MapToolEventQueue.dispatchEvent(MapToolEventQueue.java:54) [MapTool.jar:1.13.2]
        at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.run(Unknown Source) [?:?]
Caused by: java.lang.RuntimeException: java.io.IOException: Could not load image 
        at net.rptools.maptool.client.ui.theme.RessourceManager.lambda$getIcon$6(RessourceManager.java:486) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getFromHashMapsAndCache(RessourceManager.java:520) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:470) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:450) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getSmallIcon(RessourceManager.java:457) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.lookuptable.LookupTablePanel.initImportButton(LookupTablePanel.java:225) ~[MapTool.jar:1.13.2]
        ... 26 more
Caused by: java.io.IOException: Could not load image 
        at net.rptools.lib.image.ImageUtil.bytesToImage(ImageUtil.java:314) ~[MapTool.jar:1.13.2]
        at net.rptools.lib.image.ImageUtil.getImage(ImageUtil.java:98) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.lambda$getIcon$6(RessourceManager.java:480) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getFromHashMapsAndCache(RessourceManager.java:520) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:470) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:450) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getSmallIcon(RessourceManager.java:457) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.lookuptable.LookupTablePanel.initImportButton(LookupTablePanel.java:225) ~[MapTool.jar:1.13.2]
        ... 26 more
2023-08-30 08:54:59.697 ERROR net.rptools.maptool.client.swing.AbeillePanel - Could not init method: initExportButton
java.lang.reflect.InvocationTargetException: null
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
        at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at net.rptools.maptool.client.swing.AbeillePanel.panelInit(AbeillePanel.java:99) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.lookuptable.LookupTablePanel.<init>(LookupTablePanel.java:48) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.getLookupTablePanel(MapToolFrame.java:730) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.initializeFrames(MapToolFrame.java:651) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.configureDocking(MapToolFrame.java:540) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.MapToolFrame.<init>(MapToolFrame.java:451) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.MapTool.initialize(MapTool.java:665) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.MapTool.lambda$main$9(MapTool.java:1788) ~[MapTool.jar:1.13.2]
        at java.awt.event.InvocationEvent.dispatch(Unknown Source) ~[?:?]
        at java.awt.EventQueue.dispatchEventImpl(Unknown Source) ~[?:?]
        at java.awt.EventQueue$4.run(Unknown Source) ~[?:?]
        at java.awt.EventQueue$4.run(Unknown Source) ~[?:?]
        at java.security.AccessController.doPrivileged(Unknown Source) [?:?]
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:?]
        at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:?]
        at net.rptools.maptool.client.swing.MapToolEventQueue.dispatchEvent(MapToolEventQueue.java:54) [MapTool.jar:1.13.2]
        at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:?]
        at java.awt.EventDispatchThread.run(Unknown Source) [?:?]
Caused by: java.lang.RuntimeException: java.io.IOException: Could not load image 
        at net.rptools.maptool.client.ui.theme.RessourceManager.lambda$getIcon$6(RessourceManager.java:486) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getFromHashMapsAndCache(RessourceManager.java:520) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:470) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:450) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getSmallIcon(RessourceManager.java:457) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.lookuptable.LookupTablePanel.initExportButton(LookupTablePanel.java:269) ~[MapTool.jar:1.13.2]
        ... 26 more
Caused by: java.io.IOException: Could not load image 
        at net.rptools.lib.image.ImageUtil.bytesToImage(ImageUtil.java:314) ~[MapTool.jar:1.13.2]
        at net.rptools.lib.image.ImageUtil.getImage(ImageUtil.java:98) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.lambda$getIcon$6(RessourceManager.java:480) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getFromHashMapsAndCache(RessourceManager.java:520) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:470) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getIcon(RessourceManager.java:450) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.theme.RessourceManager.getSmallIcon(RessourceManager.java:457) ~[MapTool.jar:1.13.2]
        at net.rptools.maptool.client.ui.lookuptable.LookupTablePanel.initExportButton(LookupTablePanel.java:269) ~[MapTool.jar:1.13.2]
        ... 26 more

</pre>
</details>

--- a/src/main/java/net/rptools/maptool/client/ui/theme/RessourceManager.java
+++ b/src/main/java/net/rptools/maptool/client/ui/theme/RessourceManager.java
@@ -34,13 +34,13 @@ public class RessourceManager {
       new HashMap<>() {
         {
           // This icons don't exist in classic.
-          put(Icons.ACTION_EXPORT, "");
-          put(Icons.ACTION_IMPORT, "");
-          put(Icons.PROPERTIES_TABLE_ALPHABETIC, "");
-          put(Icons.PROPERTIES_TABLE_CATEGORIES, "");
-          put(Icons.PROPERTIES_TABLE_COLLAPSE, "");
-          put(Icons.PROPERTIES_TABLE_EXPAND, "");
-          put(Icons.PROPERTIES_TABLE_HIDE_DESCRIPTION, "");
+          put(Icons.ACTION_EXPORT, null);
+          put(Icons.ACTION_IMPORT, null);
+          put(Icons.PROPERTIES_TABLE_ALPHABETIC, null);
+          put(Icons.PROPERTIES_TABLE_CATEGORIES, null);
+          put(Icons.PROPERTIES_TABLE_COLLAPSE, null);
+          put(Icons.PROPERTIES_TABLE_EXPAND, null);
+          put(Icons.PROPERTIES_TABLE_HIDE_DESCRIPTION, null);
 
           put(Icons.ACTION_CANCEL, IMAGE_DIR + "cancel_sm.png");
           put(Icons.ACTION_CLOSE, IMAGE_DIR + "collapse.png");
@@ -509,6 +509,8 @@ public class RessourceManager {
     for (var map : maps) {
       if (map.containsKey(key)) {
         ressourcePath = map.get(key);
+      }
+      if (ressourcePath != null) {
         break;
       }
     }
@@ -544,7 +546,11 @@ public class RessourceManager {
 
   public static void checkMissingFiles() {
     String basedir = "C:\\Users\\tkunze\\Source\\maptool\\src\\main\\resources\\";
-    for (String value : Set.copyOf(images.values())) {
+    for (String value : images.values()) {
+      if (value == null) {
+        continue;
+      }
+
       var source = Path.of(basedir, value);
       var target = Path.of(basedir, IMAGE_DIR, "images", source.getFileName().toString());
 
@@ -552,14 +558,22 @@ public class RessourceManager {
         System.out.println(value + " is missing!");
       }
     }
-    for (String value : Set.copyOf(classicIcons.values())) {
+    for (String value : classicIcons.values()) {
+      if (value == null) {
+        continue;
+      }
+
       var source = Path.of(basedir, value);
 
       if (Files.notExists(source)) {
         System.out.println(value + " is missing!");
       }
     }
-    for (String value : Set.copyOf(rodIcons.values())) {
+    for (String value : rodIcons.values()) {
+      if (value == null) {
+        continue;
+      }
+
       var source = Path.of(basedir, value);
 
       if (Files.notExists(source)) {

